airbyte-integrations/connectors/source-quickbooks/source_quickbooks/manifest.yaml

```diff
@@ -1,9 +1,6 @@
 version: "0.34.0"
```

Comment:
The platform actually actively tries to run the CDK that is defined by this version, so it should in general match what you have in pyproject.toml